sentry/src/db/event_aggregate.rs

```diff
@@ -52,3 +86,88 @@ pub async fn list_event_aggregates(
 
     Ok(event_aggregates)
 }
+
+#[derive(Debug)]
+struct EventData {
+    id: String,
+    event_type: String,
+    earner: Option<String>,
```

Comment:
We have `ToSql` for `ValidatorId` why not use it for this struct as well?

Reply:
Because `Postgres` doesn't recognize`ValidatorId` or `ChannelId`